zerver/openapi/zulip.yaml

```diff
@@ -17951,6 +17979,15 @@ components:
             owner_id:
               nullable: true
             services: {}
+            is_active:
+              type: boolean
+              description: |
+                A boolean describing whether the user account has been deactivated.
+
+                **Changes**: New in Zulip 8.0 (feature level 221). Previously
+                we sent `realm_user` event with `op` field set to `remove`
+                when deactivating a bot and `realm_user` event with `op`
+                field set to `add` when reactivating a bot.
```

Comment:
We currently use `BasicBot` reference only for the `realm_bot/update` event, so this "Changes" entry is fine but we might need to update it if we use this somewhere else in future.

Reply:
Sounds good.